.github/workflows/combine_nb_to_docs_testing.sh

```diff
@@ -31,22 +27,21 @@ SED=`which -a gsed sed | head -n1`
 $SED -i "/id\"\:/d" all_tools.ipynb langchain_quickstart_colab.ipynb llama_index_quickstart_colab.ipynb no_framework_quickstart_colab.ipynb
 
 ## Remove ipynb JSON calls
-$SED -i "/JSON/d" quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py 
+$SED -i "/JSON/d" langchain_quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py 
 ## Replace jupyter display with python print
-$SED -i "s/display/print/g" quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
+$SED -i "s/display/print/g" langchain_quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
 ## Remove cell metadata
-$SED -i "/\# In\[/d" quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
+$SED -i "/\# In\[/d" langchain_quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
 ## Remove single # lines
-$SED -i "/\#$/d" quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
+$SED -i "/\#$/d" langchain_quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
 ## Collapse multiple empty line from sed replacements with a single line
-$SED -i -e "/./b" -e ":n" -e "N;s/\\n$//;tn" quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
+$SED -i -e "/./b" -e ":n" -e "N;s/\\n$//;tn" langchain_quickstart.py llama_index_quickstart.py no_framework_quickstart.py all_tools.py
 
 # Move all generated files to the generated_files folder
 mv README.md ../../trulens_eval/README.md
 mv TOP_README.md ../../README.md
 
+mv *langchain_quickstart_colab.ipynb ../../trulens_eval/examples/colab/quickstarts/
 mv llama_index_quickstart.py ../../trulens_eval/examples/llama_index_quickstart.py
 mv no_framework_quickstart.py ../../trulens_eval/examples/no_framework_quickstart.py
-mv *.py ../../trulens_eval/examples/
-mv *quickstart_colab.ipynb ../../trulens_eval/examples/colab/quickstarts/
-mv all_tools* ../../trulens_eval/generated_files/
```

Comment:
why are these removed? This is does our testing, colab notebooks, and non-colab examples

Reply:
Undid these changes